scripts/paper/paper_preflight.py

```
@@ -208,6 +209,283 @@ def check_appendix_source_note(repo_root: pathlib.Path, findings: Findings) -> N
+                break
+            continue
+        in_table = True
+        cells = [cell.strip() for cell in stripped.strip("|").split("|")]
+        rows.append(cells)
+    return rows
+
+
+def normalize_table_header(cell: str) -> str:
+    normalized = cell.strip().strip("`").lower()
+    normalized = re.sub(r"\s+", " ", normalized)
+    return normalized
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `normalized` before `return` statement

Remove unnecessary assignment

(RET504)